src/editor/Renderer.cpp

```diff
@@ -1295,9 +1362,11 @@ void Renderer::pickObject() {
 	int oldEntIdx = pickInfo.entIdx;
 	memset(&pickInfo, 0, sizeof(PickInfo));
 	pickInfo.bestDist = FLT_MAX;
+
 	for (int i = 0; i < mapRenderers.size(); i++) {
+		mapRenderers[i]->preRenderEnts();

```

Comment:
this is kind of expensive to do on every selection. Why is it needed?

Reply:
@wootguy @wootguy i don't know why this code added, many time elapsed from adding..., I can remove it ?